smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -816,7 +817,7 @@ private sendBasicEvents(device, param, value) {
 
 	switch (param) {
 		case "on":
-			device.sendEvent(name: "switch", value: (value == true) ? "on" : "off")
+			device.sendEvent(name: "switch", value: (value == true) ? "on" : "off", isStateChange: true)
```

Comment:
If I dont, I dont think lastactivity will update since cloud will not send event if state is the same